test/FsAutoComplete.Tests.Lsp/Helpers.fs

```diff
@@ -415,13 +460,11 @@ let dotnetToolRestore dir =
 let serverInitialize path (config: FSharpConfigDto) createServer =
   async {
     dotnetCleanup path
-    let files = Directory.GetFiles(path)
 
-    if files |> Seq.exists (fun p -> p.EndsWith ".fsproj") then
-      do! dotnetRestore path
+    for file in System.IO.Directory.EnumerateFiles(path, "*.fsproj", SearchOption.AllDirectories) do
+      do! file |> Path.GetDirectoryName |> dotnetRestore
```

Comment:
We weren't doing restore properly for anything that contained subfolders with fsprojs